src/Error/Renderer/ConsoleExceptionRenderer.php

```diff
@@ -102,11 +101,13 @@ public function render()
     /**
      * Write output to the output stream
      *
-     * @param string $output The output to print.
+     * @param \Psr\Http\Message\ResponseInterface|string $output The output to print.
      * @return void
      */
-    public function write($output): void
+    public function write(ResponseInterface|string $output): void
     {
-        $this->output->write($output);
+        if (is_string($output)) {
+            $this->output->write($output);
+        }
```

Comment:
@markstory We can't narrow the parameter type. Not sure what `ResponseInterface` is meant to do here.

Reply:
ResponseInterface won't ever happen in a console environment.

Reply:
This is where using `asset()` would make sense.